Don't install garbage that may be present in sources (e.g. .orig patch files)

--- CMakeLists.txt~	2014-06-25 11:07:52.000000000 +0200
+++ CMakeLists.txt	2018-08-22 11:56:19.011092500 +0200
@@ -31,7 +31,8 @@
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
 install(DIRECTORY include/class_loader/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.h")
 
 if(CATKIN_ENABLE_TESTING)
   add_subdirectory(test)
